packages/contracts/test/Attesting.test.ts

```diff
@@ -395,4 +395,100 @@ describe('Attesting', () => {
             await hardhatEthers.provider.getBalance(unirepContract.address)
         ).to.equal(0)
     })
+
+    it('gst attestation from unregistered attester should fail', async () => {
+        let nonAttester = accounts[5]
```

Comment:
I think we should use `const` keyword more than `let`.

Reply:
Definitely, not sure how all these slipped through